tests/test_rules.py

```diff
@@ -79,7 +79,7 @@ def test_legal_trademark_violations(self):
     def test_optional_tags(self):
         files_with_incorrect_tags = []
         tags_pattern = re.compile(
-            r"cve\.\d+\.\d+|attack\.t\d+\.*\d*|attack\.[a-z_]+|car\.\d{4}-\d{2}-\d{3}")
+            r"cve\.\d+\.\d+|(attack\.t\d{4}\.\d{3}$|attack\.t\d{4}$)|attack\.[a-z_]+|car\.\d{4}-\d{2}-\d{3}")
```

Comment:
attack\.t\d{4}\.\d{3}$|attack\.[gts]\d{4}$

Reply:
All of it is fine in my opinion. We can also add the groups and ?software?

Reply:
I will add the `g` and `s` yes